### ISSUE TITLE
2.0.23 breaks on routes with url params

### ISSUE THREAD

**Issue by jpengelbrecht** [harlan-zw/nuxt-og-image#78]
### Describe the bug

![image](https://github.com/harlan-zw/nuxt-og-image/assets/93583336/d7e14bf1-ea2a-4933-964c-13818d6894c8)

### System / Nuxt Info

```Shell
------------------------------
- Operating System: Darwin
- Node Version:     v18.10.0
- Nuxt Version:     3.7.0
- CLI Version:      3.7.3
- Nitro Version:    2.6.2
- Package Manager:  npm@8.19.2
- Builder:          -
- User Config:      modules, site, sitemap, robots, ogImage, linkChecker, head, experimental, routeRules, extends, typescript, sourcemap, devServer, htmlValidator, image, runtimeConfig, schemaOrg, nitro, appConfig, ignore, hooks, css, vite
- Runtime Modules:  @pinia/nuxt@0.4.11, @nuxt/content@2.7.2, @nuxt/image-edge@1.0.0-rc.1-28164352.ad7c010, @vueuse/motion/nuxt@2.0.0, nuxt-calendly@0.1.14
- Build Modules:    -
------------------------------
```

**Comment by harlan-zw** [comment]
Thanks for the issue, should be fixed in 2.0.24

**Comment by jpengelbrecht** [comment]
@harlan-zw  legendary turn around time!  thanks for the fix!
